### ISSUE TITLE
Failing CRRao Tests

### ISSUE THREAD

**Issue by ShouvikGhosh2048** [xKDR/CRRao.jl#123]
The CRRao Tests are failing due to a `ADGradient` error.

A similar error occurs in this program:
```Julia
using Turing, StableRNGs

@model function example(x)
    m ~ Normal(0, 1)
    x ~ Normal(m, sqrt(1))
end

res = sample(StableRNG(123), example(1), NUTS(), 10)
```

This issue might be related to [this issue](https://github.com/TuringLang/Turing.jl/issues/2007).

**Comment by ayushpatnaikgit** [comment]
Seems to be working on Turing v0.26.0, which version are you using?

**Comment by ShouvikGhosh2048** [comment]
I was using v0.23.3. That might have been the issue. CRRao also currently uses v0.23.3 which might be causing it's test to fail.

**Comment by ayushpatnaikgit** [comment]
Okay, I've removed the old version of Turing in this PR. 
https://github.com/xKDR/CRRao.jl/pull/121

Let's wait for the CI to complete.

**Comment by ayushpatnaikgit** [comment]
These two logistic regression tests are failing
```
(prior, link) = (Prior_HorseShoe(), Logit(CRRao.Logit_Link))
(prior, link) = (Prior_HorseShoe(), Probit(CRRao.Probit_Link))
```
For Negative binomial, we get an error outside the test: 
`zero(p) < p <= one(p)`

These are the only two errors.

**Comment by sourish-cmi** [comment]
It is being fixed and new release  `0.1.1` is being done
